Subject: [PATCH] floppy build fix
Date:	Sun, 02 Jan 2005 17:46:55 +0000
X-BK-ChangeSetKey: akpm@osdl.org[torvalds]|ChangeSet|20050102174655|48099
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2113, 2005/01/02 09:46:55-08:00, akpm@osdl.org

	[PATCH] floppy build fix
	
	drivers/block/floppy.c: In function `init_module':
	drivers/block/floppy.c:4598: error: parse error before "UTS_RELEASE"
	
	Not sure what went wrong here - just kill the thing.
	
	Signed-off-by: Andrew Morton <akpm@osdl.org>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 floppy.c |    2 --
 1 files changed, 2 deletions(-)


diff -Nru a/drivers/block/floppy.c b/drivers/block/floppy.c
--- a/drivers/block/floppy.c	2005-01-02 10:10:17 -08:00
+++ b/drivers/block/floppy.c	2005-01-02 10:10:17 -08:00
@@ -4595,8 +4595,6 @@
 
 int init_module(void)
 {
-	printk(KERN_INFO "inserting floppy driver for " UTS_RELEASE "\n");
-
 	if (floppy)
 		parse_floppy_cfg_string(floppy);
 	return floppy_init();
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

